addons/mail/models/fetchmail.py

```diff
@@ -250,8 +253,11 @@ def fetch_mail(self):
                         if num_messages < MAX_POP_MESSAGES or failed_in_loop == num:
                             break
                         pop_server.quit()
-                except Exception:
-                    _logger.info("General failure when trying to fetch mail from %s server %s.", server.server_type, server.name, exc_info=True)
+                except Exception as e:
+                    if raise_exception:
+                        raise ValidationError(_("Couldn't get your emails. Check out the error message below for more info:\n%s", e)) from e
```

Comment:
@std-odoo But wait, this require export translation right, do i need to that here and for all the languages as well ?

Reply:
no, it's not needed when we target master :)

Reply:
:smiley: Ok thank you